src/adaptors/http2/http2_adaptor.c

```diff
@@ -2699,13 +2467,13 @@ static void clean_conn_buffs(qdr_http2_connection_t* conn)
     //
     // Free all the buffers on this session. This session is closed and any unsent buffers should be freed.
     //
-    qd_adaptor_buffer_t *buf      = DEQ_HEAD(conn->buffs);
+    qd_adaptor_buffer_t *buf      = DEQ_HEAD(conn->out_buffs);
```

Comment:
Suggestion: use the adaptor buffer utility qd_adaptor_buffer_list_free_buffers() - it will be faster.

Reply:
Done, thanks